source/math/FloatSize3D.ooc

```diff
@@ -33,8 +33,8 @@ FloatSize3D: cover {
 	init: func@ ~default { this init(0.0f, 0.0f, 0.0f) }
 	pNorm: func (p: Float) -> Float {
 		p == 1 ?
-		this width abs() + this height abs() :
-		(this width abs() pow(p) + this height abs() pow(p)) pow(1 / p)
+		this width abs() + this height abs() + this depth abs() :
+		(this width abs() pow(p) + this height abs() pow(p) + this depth abs() pow(p)) pow(1.0f / p)
```

Comment:
On the other hand, how likely is `p` to be `1.0f`? I think maybe this whole function should just be reduced to
```ooc
(this width abs() pow(p) + this height abs() pow(p) + this depth abs() pow(p)) pow(1.0f / p)
```
but if `1.0f` is a very common value for `p`, the performance boost might make up for the decreased readability.